src/main/java/eiteam/esteemedinnovation/materials/refined/ItemMetalNugget.java

```diff
@@ -13,10 +14,11 @@ public ItemMetalNugget() {
     }
 
     @Override
-    @SideOnly(Side.CLIENT)
-    public void getSubItems(Item item, CreativeTabs tabs, NonNullList<ItemStack> subItems) {
-        for (Types type : Types.values()) {
-            subItems.add(new ItemStack(this, 1, type.getMeta()));
+    public void getSubItems(CreativeTabs tab, NonNullList<ItemStack> items) {
+        if(tab == EsteemedInnovation.tab) {
```

Comment:
Missing space after `if`

Reply:
Come to think of it, what is the point of this if statement in all of these item classes?

Reply:
Starting in 1.12 (I think) unless we have that if statement there, all of the different subitems will end up in every single creative tab.